<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
        <class>SplashScreenDialog</class>
        <widget class="QDialog" name="SplashScreenDialog">
                <property name="geometry">
                        <rect>
                                <x>0</x>
                                <y>0</y>
                                <width>571</width>
                                <height>332</height>
                        </rect>
                </property>
                <property name="windowTitle">
                        <string>TravelTime</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                        <property name="spacing">
                                <number>0</number>
                        </property>
                        <property name="leftMargin">
                                <number>0</number>
                        </property>
                        <property name="topMargin">
                                <number>0</number>
                        </property>
                        <property name="rightMargin">
                                <number>0</number>
                        </property>
                        <property name="bottomMargin">
                                <number>0</number>
                        </property>
                        <item>
                                <widget class="QWidget" name="widget_2" native="true">
                                        <property name="styleSheet">
                                                <string notr="true">background-color: rgba(19,28,76);</string>
                                        </property>
                                        <layout class="QVBoxLayout" name="verticalLayout">
                                                <item>
                                                        <widget class="QLabel" name="label_3">
                                                                <property name="maximumSize">
                                                                        <size>
                                                                                <width>500</width>
                                                                                <height>97</height>
                                                                        </size>
                                                                </property>
                                                                <property name="text">
                                                                        <string/>
                                                                </property>
                                                                <property name="pixmap">
                                                                        <pixmap>../resources/banner_splashscreen.png</pixmap>
                                                                </property>
                                                                <property name="scaledContents">
                                                                        <bool>true</bool>
                                                                </property>
                                                        </widget>
                                                </item>
                                                <item>
                                                        <widget class="QTextBrowser" name="textBrowser">
                                                                <property name="frameShape">
                                                                        <enum>QFrame::NoFrame</enum>
                                                                </property>
                                                                <property name="lineWidth">
                                                                        <number>0</number>
                                                                </property>
                                                                <property name="html">
                                                                        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'sans-serif'; font-size:11pt; font-weight:600; color:#ffffff;&quot;&gt;TravelTime plugin for QGIS&lt;/span&gt;&lt;span style=&quot; font-family:'sans-serif'; font-size:11pt; color:#ffffff;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'sans-serif'; font-size:11pt; color:#ffffff;&quot;&gt;The TravelTime API helps users find locations by journey time rather than using &lt;/span&gt;&lt;span style=&quot; font-family:'sans-serif'; font-size:11pt; font-style:italic; color:#ffffff;&quot;&gt;as the crow flies&lt;/span&gt;&lt;span style=&quot; font-family:'sans-serif'; font-size:11pt; color:#ffffff;&quot;&gt; distance. Time-based searching gives users more opportunities for personalisation and delivers a more relevant search. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'sans-serif'; font-size:11pt; color:#ffffff;&quot;&gt;The plugin adds a toolbar offering easy access to all features. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'sans-serif'; font-size:11pt; color:#ffffff;&quot;&gt;Please click on the help button to help you get started.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                                                                </property>
                                                        </widget>
                                                </item>
                                        </layout>
                                </widget>
                        </item>
                        <item>
                                <widget class="QWidget" name="widget" native="true">
                                        <layout class="QHBoxLayout" name="horizontalLayout">
                                                <item>
                                                        <widget class="QCheckBox" name="dontShowAgainCheckBox">
                                                                <property name="text">
                                                                        <string>Don't show again</string>
                                                                </property>
                                                        </widget>
                                                </item>
                                                <item>
                                                        <widget class="QDialogButtonBox" name="buttonBox">
                                                                <property name="orientation">
                                                                        <enum>Qt::Horizontal</enum>
                                                                </property>
                                                                <property name="standardButtons">
                                                                        <set>QDialogButtonBox::Ok</set>
                                                                </property>
                                                        </widget>
                                                </item>
                                        </layout>
                                </widget>
                        </item>
                </layout>
        </widget>
        <resources/>
        <connections>
                <connection>
                        <sender>buttonBox</sender>
                        <signal>accepted()</signal>
                        <receiver>SplashScreenDialog</receiver>
                        <slot>accept()</slot>
                        <hints>
                                <hint type="sourcelabel">
                                        <x>248</x>
                                        <y>254</y>
                                </hint>
                                <hint type="destinationlabel">
                                        <x>157</x>
                                        <y>274</y>
                                </hint>
                        </hints>
                </connection>
                <connection>
                        <sender>buttonBox</sender>
                        <signal>rejected()</signal>
                        <receiver>SplashScreenDialog</receiver>
                        <slot>reject()</slot>
                        <hints>
                                <hint type="sourcelabel">
                                        <x>316</x>
                                        <y>260</y>
                                </hint>
                                <hint type="destinationlabel">
                                        <x>286</x>
                                        <y>274</y>
                                </hint>
                        </hints>
                </connection>
        </connections>
</ui>
